.github/workflows/build.yml

```diff
@@ -171,27 +171,50 @@ jobs:
             io.artifacthub.package.readme-url=https://raw.githubusercontent.com/${{ github.repository }}/main/README.md
             io.artifacthub.package.logo-url=https://avatars.githubusercontent.com/u/120078124?s=200&v=4
 
+      # Workaround to fix indentation for the labels so that retries work.
```

Comment:
Can you help me understand why this left padding is required?

Reply:
I see you have comments about the white space issue already.

Reply:
Without it, it evalutes the YAML to 

```
with:
    action: redhat-actions/buildah-build@v2
    with: containerfiles: |
    ./Containerfile
  image: kinoite-main
  tags: |
    pr-503-39 812996e-39 pr-503 812996e
  build-args: |
    IMAGE_NAME=kinoite
    SOURCE_ORG=fedora-ostree-desktops
    SOURCE_IMAGE=kinoite
    FEDORA_MAJOR_VERSION=39
    RPMFUSION_MIRROR=
  labels: io.artifacthub.package.logo-url=https://avatars.githubusercontent.com/u/120078124?s=200&v=4
  io.artifacthub.package.readme-url=https://raw.githubusercontent.com/ublue-os/main/main/README.md
  org.opencontainers.image.created=2024-02-18T09:21:11.721Z
  org.opencontainers.image.description=A base Universal Blue kinoite image with batteries included
  org.opencontainers.image.licenses=Apache-2.0
  org.opencontainers.image.revision=812996efb45acbbebbb18ac9ec5[67](https://github.com/ublue-os/main/actions/runs/7947996260/job/21697505580#step:8:68)a3fe[79](https://github.com/ublue-os/main/actions/runs/7947996260/job/21697505580#step:8:80)289fa
  org.opencontainers.image.source=https://github.com/ublue-os/main
  org.opencontainers.image.title=kinoite-main
  org.opencontainers.image.url=https://github.com/ublue-os/main
  org.opencontainers.image.version=39.20240218.0
```

This is not valid YAML, and breaks without valid indentation. Sadly, YAML is very sensitive to indentation, unlike JSON. By adding two spaces to the front of each line in that env variable and then passing it in, it is properly indented.